src/Components/SearchResults.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import TableRow from './TableRow'
+import NewBookingForm from './NewBookinForm'
+import TableColumn from './TableColumn';
+class SearchResults extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { results: null };
+    }
+    componentDidMount() {
+        this.setState(previousState => {
+            return { results: this.props.results }
+        })
+    }
+    componentWillReceiveProps(nextProps) {
+        //Perform some operation
+        this.setState({ results: nextProps.results });
+    }
+
+    sortAndUpdate = (nameOFCoulumn, clicked) => {
+        console.log(nameOFCoulumn)
+        const ascendingSort = (a, b) => {
+            console.log(a.firstName)
+            if (nameOFCoulumn === "First Name") {
+                if (a.firstName < b.firstName) { return -1; }
+                if (a.firstName > b.firstName) { return 1; }
```

Comment:
This method repeats itself a bit. How could we clean it up so that it is more DRY?